consensus/ibft/snapshot.go

```diff
@@ -21,7 +21,7 @@ func (i *Ibft) setupSnapshot() error {
 
 	// Read from storage
 	if i.config.Path != "" {
-		if err := i.store.loadFromPath(i.config.Path); err != nil {
+		if err := i.store.loadFromPath(i); err != nil {
```

Comment:
Please change the method to accept the path and the logger, no need to accept the entire `Ibft` reference

Reply:
Fixed with commit b0bf7cd004339996424fc7301b460a33873b367f

Reply:
Please change out the `loadFromPath` method not to accept a spread, but instead a single logger instance

Reply:
Fixed with commit 576cbe70a70e335a395994055be9715d8c609c27